README.md

```diff
@@ -1 +1,215 @@
-# Awesome-books-ES6
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="bese_logo.jpg" alt="logo" width="200"  height="auto" />
+  <br/>
+
+  <h3><b>README</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [Screenshot](#screenshot)
+  - [Loom Presentation Video](#loom-video)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Awesome Books <a name="about-project"></a>
+
+A website for storing and retrieving books
+
+** [Awesome-books-ES6]** is a project(website) that Adds, removes and updates book information. the project was built using html, css, and js languages.
+
+## 🛠 Built With <a name="built-with">HTML,CSS and Javascript</a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://en.wikipedia.org/wiki/HTML">HTML</a></li>
+    <li><a href="https://www.w3.org/Style/CSS/Overview.en.html">CSS</a></li>
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="">N/A</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="">N/A</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[HTML]**
+- **[CSS]**
+- **[Javascript]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- The NAMM SHOW EAST AFRICA WEBSITE SCREENSHOT -->
+
+## 🚀 Screenshot <a name="screenshot"></a>
+
+![image](https://user-images.githubusercontent.com/23375153/217803810-3a3d8691-267c-4303-a68a-76139ddc6d2f.png)
+
```

Comment:
- [OPTIONAL]  Kindly make sure to add the proper screenshot of this website. The one you have is for another project. Kindly fix that.

Reply:
Oh...ok my bad... I'll surely fix that...thanks for the feedback. 👍